crates/turborepo-lib/src/shim/parser.rs

```diff
@@ -56,6 +56,14 @@ pub struct ShimArgs {
 
 impl ShimArgs {
     pub fn parse() -> Result<Self, Error> {
+        let invocation_dir = AbsoluteSystemPathBuf::cwd()?;
```

Comment:
We make this call here so testing is easier. It also means we avoid calling `std::env::current_dir` once for each `--cwd` flag and reuse this value. 

The only behavior change from this move is that if you:
 - pass the args `turbo --cwd`
 - Execute `turbo` in a directory where `current_dir` would fail (process doesn't have permission to access cwd or cwd doesn't exist)
Then we will now report the `current_dir` failure instead of complaining that you didn't pass a value for the `--cwd` flag.